CodePushPackage.m

```diff
@@ -149,50 +149,51 @@ + (NSString *)getPackageFolderPath:(NSString *)packageHash
 }
 
 + (void)downloadPackage:(NSDictionary *)updatePackage
-                            error:(NSError **)error
+       progressCallback:(void (^)(long, long))progressCallback
+           doneCallback:(void (^)())doneCallback
+           failCallback:(void (^)(NSError *err))failCallback
 {
     NSString *packageFolderPath = [self getPackageFolderPath:updatePackage[@"packageHash"]];
     
+    NSError *error;
```

Comment:
So according to https://developer.apple.com/library/ios/releasenotes/ObjectiveC/RN-TransitioningToARC/Introduction/Introduction.html#//apple_ref/doc/uid/TP40011226-CH1-SW5, 

Since ARC is turned on for us, the value will be initialized to nil.

However, I think I will still set it to nil explicitly, so that future contributors won't have to go through this investigation. Seems like many people think it is good practice to do so anyway too. Thanks :)